src/targets/happymodel_2400.ini

```diff
@@ -0,0 +1,35 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:HappyModel_ES24TX_2400_TX_via_UART]
+extends = env:DIY_2400_TX_ESP32_SX1280_E28_via_UART
+
+[env:HappyModel_ES24TX_2400_TX_via_WIFI]
+extends = env:HappyModel_ES24TX_2400_TX_via_UART
+
+
+# ********************************
+# Receiver targets
+# ********************************
+
+[env:HappyModel_EP_2400_RX_via_UART]
+extends = env:DIY_2400_RX_ESP8285_SX1280_via_UART
+
+[env:HappyModel_EP_2400_RX_via_BetaflightPassthrough]
+extends = env:HappyModel_EP_2400_RX_via_UART
```

Comment:
Should either extend `DIY_2400_RX_ESP8285_SX1280_via_BetaflightPassthrough` and remove the upload parts or change the `upload_command` to be the same as current develop branch as there have been changes in this area.

Reply:
good point. I think it is enough to just extend `DIY_2400_RX_ESP8285_SX1280_via_BetaflightPassthrough` as is.

Reply:
Done.